apps/relayer/ts/message/message.dto.ts

```diff
@@ -17,7 +19,7 @@ import { PublicKeyValidator } from "./validation";
 /**
  * Max messages per batch
  */
-const MAX_MESSAGES = 20;
+export const MAX_MESSAGES = 20;
```

Comment:
should we make this configurable via ENV variables, or maybe pull from contract's batch size instead?